recipes/r-nanopore/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = '0.2.5' %}
+{% set version = '0.2.7' %}
```

Comment:
should this also indicate the b version?

Reply:
I don't think so - there is one character in the R package versioning that has been fixed ... there will be no confusion between versions and the bioconda version should ideally be in sync with the R package version ...

Reply:
versions updated across all objects and documents - license file returned ...